docs/report.md

```diff
@@ -40,4 +40,8 @@ Here comes a description of our domain model.
 
 ## License
 
-## LLMs, ChatGPT, CoPilot, and others
\ No newline at end of file
+## LLMs, ChatGPT, CoPilot, and others
+A large level model is used for language understanding and generation. We have used ChatGPT and Co-Pilot in this project. We used these to expedite the coding in certain areas. 
+We tried asking ChatGPT when hitting an error that we could not figure out why we recieved. ChatGPT could actually help finding the error sometimes, making this valueable to us.
```

Comment:
Jeg synes også man kunne komlimentere Chatten for at være god til at gennemskue sammenhænge over flere domæner. Dokumentation der beskriver domæner (Azure, Github, ASP.NET) beskriver jo det enkelte domæne,  og selvom der ofte kan findes guides til hvordan man intergere andre domæner, rammer det ikke altid lige den situation man selv sidder i. Der sparre chatten os en del hovedpine i at skulle forstå dokumentation for flere domæner til bunds for at intergere dem med hinanden, da den jo er skab til at finde sammenhænge mellem dokumentation.